glade/prefs.ui

```diff
@@ -593,6 +599,64 @@
                 <property name="left-attach">1</property>
                 <property name="top-attach">1</property>
               </packing>
+            </child>
+                        <child>
```

Comment:
What's going on with formatting here? Is Glade doing this?

Reply:
Done.